action.yml

```
@@ -139,7 +139,8 @@ runs:
   - name: Install Python 3
     if: steps.pre-installed-python.outputs.python-path == ''
     id: new-python
-    uses: actions/setup-python@v5
+    # v5.6.0
+    uses: actions/setup-python@a26af69be951a213d495a4c3e4e4022e16d87065
```

The style that's also used by renovated is having the version comment on the same line as the hash, but then yamllint complains that the line is too long:

```suggestion
    uses: actions/setup-python@a26af69be951a213d495a4c3e4e4022e16d87065  # v5.6.0
```

---

dependabot also uses this style and will update the comment if it updates the hash

is there a way to silence the lint?

---

Does this really matter here though?

---

> is there a way to silence the lint?

yamllint has a mechanism for ignores.

---

@konstin I had some success with `# yamllint disable-line rule:line-length` in the past could you try appending it? Let's have the inline version comment because Dependabot would understand that. If it doesn't work, there's also a way to disable and then re-enable the rule.

---

done